services/net/reporting/ReportingManager.cs

```diff
@@ -366,44 +379,57 @@ private async Task GenerateReportAsync(ReportRequestModel request, API.Areas.Ser
             OwnerId = request.RequestorId ?? report.OwnerId,
             PublishedOn = DateTime.UtcNow,
             Subject = subject,
-            Body = body,
+            Body = linkOnlyFormatBody,
         };
         var model = new API.Areas.Services.Models.ReportInstance.ReportInstanceModel(instance, _serializationOptions);
         var instanceModel = request.GenerateInstance ? (await this.Api.AddReportInstanceAsync(model)
             ?? throw new InvalidOperationException("Report instance failed to be returned by API")) : null;
 
-        if (request.SendToSubscribers && (to.Any() || !String.IsNullOrEmpty(request.To)))
+        if (request.SendToSubscribers)
         {
-            try
-            {
-                // Send the email.
-                var response = await SendEmailAsync(request, to, subject, body, $"{report.Name}-{report.Id}");
+            var failed = false; // I don't like this implementation because one group of emails could be successful, but the other fail.
```

Comment:
you could implement a mechanism to set a "reasonable" delay on the first email,  capture the id of the first email request and if you encounter a failure sending the second email, you call ches to cancel the first one